libs/errors/src/exception.cpp

```diff
@@ -140,23 +140,23 @@ namespace hpx {
 
     static custom_exception_info_handler_type custom_exception_info_handler;
 
-    HPX_EXPORT void set_custom_exception_info_handler(
+    void set_custom_exception_info_handler(
```

Comment:
Are these symbols marked as exported at declaration?

Reply:
That's the intention. I need verify this still (I just started off by removing most of the `HPX_EXPORT`s in cpp files, but I expect to need to add some back).